PostProcess/ScreenSpaceReflection/src/ScreenSpaceReflection.cpp

```diff
@@ -298,6 +317,7 @@ void ScreenSpaceReflection::Execute(const RenderAttributes& RenderAttribs)
     DEV_CHECK_ERR(RenderAttribs.pMaterialBufferSRV != nullptr, "RenderAttribs.pMaterialBufferSRV must not be null");
     DEV_CHECK_ERR(RenderAttribs.pMotionVectorsSRV != nullptr, "RenderAttribs.pMotionVectorsSRV must not be null");
     DEV_CHECK_ERR(RenderAttribs.pSSRAttribs != nullptr, "RenderAttribs.pSSRAttribs must not be null");
+    DEV_CHECK_ERR(RenderAttribs.FeatureFlag == m_FeatureFlags, "RenderAttribs.FeatureFlag must match with flag passed to method ScreenSpaceReflection::Prepare");
```

Comment:
You already pass feature flags to `Prepare` method. Having them in them second time in `RenderAttribs` only to check they are the same is redundant.
Please remove flags from the RenderAttribs.